index.html

```diff
@@ -11,26 +11,43 @@
     href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Pacifico&family=Rubik+Moonrocks&display=swap"
     rel="stylesheet" />
   <link rel="stylesheet" href="styles.css" />
-  <!-- <link rel="stylesheet" href="animations.css" /> -->
+  <link rel="stylesheet" href="animations.css" />
   <title>Portfolio Project</title>
 </head>
 
 <body>
   <header class="top-bar">
-    <h4>Welcome</h4>
-    <nav class="hamburger">
-      <img class="menu"
+    <!-- <h4>Welcome</h4> -->
+    <nav class="nav-bar">
+      <!-- <img class="menu"
         src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARMAAAC3CAMAAAAGjUrGAAAAMFBMVEX///8AAADl5eX09PTV1dUICAiurq5TU1Orq6tJSUnc3Nzt7e1ERET6+vpRUVG9vb0gCS9GAAAA8ElEQVR4nO3dOQ4CQRAEQZblPv//W2yUFsLoNSKekGb1SLPbAQAAAAAAAAAAAADfbodh9+kC9ViGXaYL1Hm6yWm6QGlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJnWdbrLBjXqd9pwuAAAAAMCvzstx1Ba3x9f09rjBJjbq0qQ0KU1Kk9KkNClNSpPSpDQpTUqT0qQ0KU1Kk9KkNClvQet9mnW5TRcAAAAAgP/d98PW6QJljy27fWlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJuWPs3LLAAAAAAAAAAAAAGAbPhzZL1DD5UvKAAAAAElFTkSuQmCC"
-        alt="Menu Icon" />
+        alt="Menu Icon" /> -->
```

Comment:
- [OPTIONAL] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)